### PR TITLE
Remove use of `var` to follow project code style

### PR DESCRIPTION
**Pull request by pmiossec**
Note: done automatically by VS (with 2 or 3 small adaptations -- code indentation and tuples not 100% well handled--)

## Test methodology 

- Manual

## Test environment(s) 

- Git Extensions 33.33.33
- Build 03e983f6479d7fafa2ea21a13281c1d6270b6794
- Git 2.42.0.windows.2
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.21
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.21 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: GitUI/GitUI.csproj [hunk 1]
```diff
@@ -67,7 +67,9 @@
 
     <!-- Settings -->
     <None Update="Properties\Settings.settings" Generator="SettingsSingleFileGenerator" LastGenOutput="Settings.Designer.cs" />
-    <Compile Update="Properties\Settings.Designer.cs" AutoGen="True" DependentUpon="Settings.settings" />
+    <Compile Update="Properties\Settings.Designer.cs" AutoGen="True" DependentUpon="Settings.settings">
+      <DesignTimeSharedInput>True</DesignTimeSharedInput>
```
Comment (gerhardol) [inline GitUI/GitUI.csproj:71]: ?

File: GitUI/GitUIExtensions.cs [hunk 2]
```diff
@@ -49,7 +49,7 @@ public static class GitUIExtensions
                 return;
             }
 
-            var firstId = item.FirstRevision?.ObjectId ?? item.SecondRevision.FirstParentId;
+            ObjectId firstId = item.FirstRevision?.ObjectId ?? item.SecondRevision.FirstParentId;
```
Comment (gerhardol) [inline GitUI/GitUIExtensions.cs:52]:
```suggestion
            ObjectId? firstId = item.FirstRevision?.ObjectId ?? item.SecondRevision.FirstParentId;
```

File: GitUI/GitUIExtensions.cs [hunk 3]
```diff
@@ -132,12 +132,12 @@ void OpenWithDiffTool()
...
                 }
 
-                var task = file.GetSubmoduleStatusAsync();
+                Task<GitSubmoduleStatus?> task = file.GetSubmoduleStatusAsync();
 
                 if (file.IsSubmodule && task is not null)
                 {
                     // Patch already evaluated
-                    var status = await task;
+                    GitSubmoduleStatus status = await task;
```
Comment (gerhardol) [inline GitUI/GitUIExtensions.cs:140]:
```suggestion
                    GitSubmoduleStatus? status = await task;
```

File: Plugins/GitUIPluginInterfaces/Settings/CredentialsSetting.cs [hunk 4]
```diff
@@ -26,14 +26,14 @@ public void SaveValue(ISettingsSource settings, string userName, string password
...
-                var currentCredentials = GetValueOrDefault(settings);
+                NetworkCredential currentCredentials = GetValueOrDefault(settings);
                 if (currentCredentials.UserName == userName && currentCredentials.Password == password)
                 {
                     return;
                 }
             }
 
-            var newCredentials = string.IsNullOrWhiteSpace(userName)
+            NetworkCredential newCredentials = string.IsNullOrWhiteSpace(userName)
```
Comment (gerhardol) [inline Plugins/GitUIPluginInterfaces/Settings/CredentialsSetting.cs:36]:
```suggestion
            NetworkCredential? newCredentials = string.IsNullOrWhiteSpace(userName)
```

**Review by gerhardol — APPROVED** [review]
@RussKie has some bigger PRs that may be affected and should comment before merge

**Comment by RussKie** [comment]
The nullable annotations should be done anyway, so taking this to avoid any conflicts.

Thank you

**Comment by pmiossec** [comment]
FYI:

Call me naive but I just noticed that VS automatic conversion of **all** the Solution has done only a very small subset (around 1/4) 😭 
I was a little surprised because I found the number of file not as big as expected but, without a real knowledge of the number of cs files, I have not thought about doing a verification......

82ujlv

😭 😭

**Comment by RussKie** [comment]
🤣

**Comment by RussKie** [comment]
There's a diagnostics that should be dialled up in the .editorconfig file.

**Pull request merged**